uiowa/uiowa
# Feature: Update SiteNow V2 Media embeds + Coronavirus split removal

This is a work in progress.

Resolves #865
Resolves #2358

# How to test

* `blt ds --site=hawkeyemarchingband.uiowa.edu` or another V2 site to test. The update hook skips non-V2 sites - presumably because there is nothing to update.
  * Example pages with different size media:
    * https://hawkeyemarchingband.local.drupal.uiowa.edu/about/history
    * https://hawkeyemarchingband.local.drupal.uiowa.edu/band-140
    * https://sitesgis.local.drupal.uiowa.edu/gis-day
    * http://brand.local.drupal.uiowa.edu/
    * http://brand.local.drupal.uiowa.edu/logo
    * https://hr.local.drupal.uiowa.edu/careers/professional-and-scientific-positions/career-development-planning
* coronvirus.local.drupal.uiowa.edu
  * no longer a hybrid v2/v3. You should see no v2 content block stuff on the page content type instead you should see a body field. You should be able to save the page without error. Any WYSIWYG placed media should be using the `<drupal-media>` markup and look "normal"
  * depending on current site activity, you may encounter a import error or a save error if there is paragraphs items or a placed field_page_content_block field on the layout. we will be manually clearing these and testing in DEV before deployment to PROD.
